test/e2e/components/vl-upload.js

```diff
@@ -41,13 +41,33 @@ class VlUpload extends VlElement {
   }
 
   async getTitle() {
-    const element = await this.shadowRoot.findElement(By.css('.vl-upload__element__button__container'));
-    return element.getText();
+    return this.shadowRoot.findElement(By.id('title')).then(async () => {
+      const element = await this.shadowRoot.findElement(By.css('.vl-upload__element__button__container'));
+      return element.getText();
+    }).catch(async () => {
+      // op firefox specifiek, duurt het één tick langer vooraleer het slot beschikbaar is
```

Comment:
@coemans  ik blijf momenteel hier nog tegen een issue lopen en zal hiervoor straks eens in bellen. Op chrome wordt de text mooi opgehaald maar op firefox lijkt het een tick langer te duren vooraleer de slot zijn content krijgt en faalt dit altijd consistent.

Reply:
nog wat verder naar gekeken maar nog geen oplossing gevonden

Reply:
bij deze aangepast en opgelost door Tom.

Build is ondertussen ook groen:
https://www.milieuinfo.be/bamboo/browse/WEBCO-VLUIUPLOAD43-17